test_rmw_implementation/test/test_init_shutdown.cpp

```diff
@@ -34,6 +35,8 @@ class CLASSNAME (TestInitShutdown, RMW_IMPLEMENTATION) : public ::testing::Test
     options = rmw_get_zero_initialized_init_options();
     rmw_ret_t ret = rmw_init_options_init(&options, rcutils_get_default_allocator());
     ASSERT_EQ(RMW_RET_OK, ret) << rcutils_get_error_string().str;
+    options.enclave = rcutils_strdup("/", rcutils_get_default_allocator());
```

Comment:
Shouldn't this allocated string be deallocated?

Reply:
It'll be taken care of by `rmw_init_options_fini()`.

Reply:
Just checked, you are right!